tests/prefetch_related/tests.py

```diff
@@ -891,6 +891,16 @@ def test_prefetch_GFK_uuid_pk(self):
         qs = Comment.objects.prefetch_related('content_object_uuid')
         self.assertEqual([c.content_object_uuid for c in qs], [article])
 
+    def test_prefetch_GFK_fk_pk(self):
+        book = Book.objects.create(title="Poems")
+        book_with_year = BookWithYear.objects.create(book=book, published_year=2019)
+        Comment.objects.create(comment="awesome", content_object=book_with_year)
+
+        with self.assertNumQueries(2):
+            qs = Comment.objects.prefetch_related("content_object")
+            for c in qs:
+                self.assertIsNotNone(c.content_object)
```

Comment:
Please chop blank line and use single quotes.

I would check `content_object` value instead of number of queries, e.g.
```python
    def test_prefetch_GFK_fk_pk(self):
        ...
        Comment.objects.create(comment='awesome', content_object=book_with_year)
        qs = Comment.objects.prefetch_related('content_object')
        self.assertEqual([c.content_object for c in qs], [book_with_year])
```